lib/rules/exports-style.js

```diff
@@ -139,6 +139,82 @@ function getExportsNodes(scope) {
     return variable.references.map(reference => reference.identifier)
 }
 
+function getReplacementForProperty(property, sourceCode) {
+    if (property.kind === "get" || property.type === "SpreadElement") {
+        // module.exports = { get foo() { ... } }
+        // module.exports = { foo }
+        // module.exports = { ...foo }
+        // We don't have a similarly nice syntax for adding these directly
+        // on the exports object. Give up on fixing the whole thing.
+        return undefined
+    }
+    let fixedValue = sourceCode.getText(property.value)
+    if (property.method) {
+        fixedValue = `function${
+            property.value.generator ? "*" : ""
+        } ${fixedValue}`
+        if (property.value.async) {
+            fixedValue = `async ${fixedValue}`
+        }
+    }
+    const comments = sourceCode.getComments(property)
```

Comment:
`sourceCode.getComments()` method has been deprecated since ESLint v4.0.0 (https://github.com/eslint/eslint/blob/caeb223c4f7b0b6fe35e5348ae0df4c6446b5bed/docs/user-guide/migrating-to-4.0.0.md#-ast-nodes-no-longer-have-comment-properties). Would you fix it with the new methods?

Reply:
Absolutely, fixed in 8d7cec4

I had no idea, I was just playing around with it in the debugger to discover how to get to the comments :)